test/utils/Deployers.sol

```diff
@@ -114,6 +114,25 @@ contract Deployers {
         return (currency0, currency1);
     }
 
+    function deployMintAndApproveCurrency() internal returns (Currency currency) {
```

Comment:
i feel like `deployMintAndApprove2Currencies` should just call this twice? to save duplication of the list of approvals needed. Then it can just set the 2 addresses returned to 0 and 1 and return them